### ISSUE TITLE
Release 2.0.7 fails to compile

### ISSUE THREAD

**Issue by oleg-tsybulsky** [electron/remote#119]
Error: node_modules/@electron/remote/index.d.ts:42:41 - error TS2551: Property 'NativeImage' does not exist on type 'typeof CrossProcessExports'. Did you mean 'nativeImage'?

42 export var nativeImage: typeof Electron.NativeImage;
                                           ~~~~~~~~~~~

  node_modules/electron/electron.d.ts:17072:11
    17072     const nativeImage: typeof NativeImage;
                    ~~~~~~~~~~~
    'nativeImage' is declared here.

Error: node_modules/@electron/remote/index.d.ts:57:41 - error TS2551: Property 'WebContents' does not exist on type 'typeof CrossProcessExports'. Did you mean 'webContents'?

57 export var webContents: typeof Electron.WebContents;
                                           ~~~~~~~~~~~

  node_modules/electron/electron.d.ts:17111:11
    17111     const webContents: typeof WebContents;
                    ~~~~~~~~~~~
    'webContents' is declared here.

Error: node_modules/@electron/remote/index.d.ts:58:42 - error TS2551: Property 'WebFrameMain' does not exist on type 'typeof CrossProcessExports'. Did you mean 'webFrameMain'?

58 export var webFrameMain: typeof Electron.WebFrameMain;
                                            ~~~~~~~~~~~~

  node_modules/electron/electron.d.ts:17115:11
    17115     const webFrameMain: typeof WebFrameMain;
                    ~~~~~~~~~~~~
    'webFrameMain' is declared here.

**Comment by johannesjo** [comment]
Should be fixed by this:
https://github.com/electron/remote/pull/111

If someone would merge it.

**Comment by andersk** [comment]
Yeah, if anyone is listening, please merge #111, and also #113 so that future problems like this will be caught in CI.

**Comment by oleg-tsybulsky** [comment]
Resolved in 2.0.8
